src/Microsoft.Identity.Client/CacheV2/TokenCacheV2.cs

```diff
@@ -147,13 +147,39 @@ public void Deserialize(byte[] unifiedState)
         }
 
         /// <inheritdoc />
-        public byte[] SerializeV3()
+        public byte[] SerializeMsalV3()
         {
             throw new NotImplementedException();
         }
 
         /// <inheritdoc />
-        public void DeserializeV3(byte[] bytes)
+        public void DeserializeMsalV3(byte[] bytes, bool merge = false)
+        {
+            throw new NotImplementedException();
+        }
+
+        /// <inheritdoc />
+        public byte[] SerializeMsalV2()
+        {
+            // Note: This is effectively current Serialize
+            throw new NotImplementedException();
```

Comment:
don't we want to wire it up?

Reply:
This is the internal V2 cache object. these are not used at this time, it's a refactoring exercise.